### PR TITLE
Work Allocation, Sprint 20.4 + 20.5

### PR DESCRIPTION
**Pull request by paul-graham**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/secure/RapidBoard.jspa?rapidView=1545&projectKey=EUI&view=reporting&chart=sprintRetrospective&sprint=7476

### Change description ###
The various tickets that make up Sprint 20.4 and Sprint 20.5 from the Work Allocation functionality.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
LGTMcp

**Review by Daniel-Lam — APPROVED** [review]
Approved on the basis of @udaydenduluri33's comment that each commit under this collation PR has already been peer reviewed and approved separately.

File: charts/xui-webapp/values.yaml [hunk 1]
```diff
@@ -17,7 +17,8 @@ nodejs:
   memoryLimits: 2048Mi
   autoscaling:        # Default is false
     enabled: true 
-    maxReplicas: 16    # Required setting
+    minReplicas: 13
+    maxReplicas: 22    # Required setting
```
Comment (udaydenduluri33) [inline charts/xui-webapp/values.yaml:21]: @ernestman28

**Comment by udaydenduluri33** [comment]
Closing this PR as PR-848 is now latest with workallocation-5

**Pull request closed without merging**